index.js

```diff
@@ -16,18 +22,60 @@ async function run() {
     const logConfigurationOptions = core.getInput("log-configuration-options", { required: false });
     const dockerLabels = core.getInput('docker-labels', { required: false });
     const command = core.getInput('command', { required: false });
+    
+     //New inputs 
+     const taskDefinitionArn = core.getInput('task-definition-arn', { required: false });
+     const taskDefinitionFamily = core.getInput('task-definition-family', { required: false });
+     const taskDefinitionRevision = Number(core.getInput('task-definition-revision', { required: false}));
+    
+
+    if (taskDefinitionFile && taskDefinitionArn) { // checks if provided both task definition and task definition arn
```

Comment:
Suggest getting rid of this comment since it is obvious what the `if` check is doing.